cms/envs/aws_migrate.py

```diff
@@ -1,30 +0,0 @@
-"""
```

Comment:
@nedbat it should be fixed in this merged PR, I believe: https://github.com/edx/configuration/pull/2723/files#diff-af208e6b12029728bf08dbe0406b5d3aR13

Or are you talking about some other configuration repo, not edx/configuration? I was unable to find any references to aws_migrate in it (even with plain text search)